automation/check-patch.e2e-kube-secondary-dns-functests.sh

```diff
@@ -29,6 +29,9 @@ main() {
 
     trap teardown EXIT
 
+    echo "Simulate network restrictions on CNAO namespace"
+    ./hack/install-network-policy.sh
```

Comment:
Please check if you can move it to `./hack/deploy-kubevirt.sh` with [[ if $CI == true ]]
this way it is one place for all the lanes